src/ert/scheduler/job.py

```diff
@@ -87,13 +95,15 @@ def running_duration(self) -> float:
             return time.time() - self._start_time
         return 0
 
-    async def _submit_and_run_once(self, sem: asyncio.BoundedSemaphore) -> None:
+    async def _submit_and_run_once(
+        self, sem: asyncio.BoundedSemaphore, driver: Driver
+    ) -> None:
```

Comment:
is this driver arg related to the commit message, or is it some other refactoring?

Reply:
It's part of the commit. The `Job` object obtains a `Driver` only when it is supposed to use it, not before.